custom_components/elegoo_printer/elegoo_sdcp/debug.py

```
@@ -20,10 +20,12 @@
         connected = await elegoo_printer.connect_printer()
         if connected:
             logger.debug("Polling Started")
-            elegoo_printer.set_printer_video_stream(toggle=True)
+            await asyncio.sleep(2)
+            logger.debug(elegoo_printer.get_current_print_thumbnail())
             while True:
-                await asyncio.sleep(2)
-                elegoo_printer.get_printer_status()
+                await asyncio.sleep(0)
+                # elegoo_printer.get_printer_status()
```

Ruff (0.8.2)

Found commented-out code

Remove commented-out code

(ERA001)